includes/core/server/request/class-get-reporting-payment-activity.md

```diff
@@ -0,0 +1,37 @@
+# `Get_Reporting_Payment_Activity` request class
+
+[ℹ️ This document is a part of __WooCommerce Payments Server Requests__](../requests.md)
+
+## Description
+
+The `WCPay\Core\Server\Request\Get_Reporting_Payment_Activity` class is used to construct the request for retrieving payment activity.
+
+## Parameters
+
+| Parameter   | Setter                                    | Immutable | Required | Default value |
+|-------------|-------------------------------------------|:---------:|:--------:|:-------------:|
+| `date_start`| `set_date_start( string $date_start )`    |    No     |    Yes   |       -       |
+| `date_end`  | `set_date_end( string $date_end )`        |    No     |    Yes   |       -       |
+| `timezone`  | `set_timezone( string $timezone )`        |    No     |    Yes   |       -       |
+
+The `date_start` and `date_end` parameters should be in the 'YYYY-MM-DDT00:00:00' format.
```

Comment:
nit: Actually, the values the server accepts can be anything that is parsable by [strtotime()](https://www.php.net/manual/en/function.strtotime.php).

Leaving it as is is fine too.

Reply:
Do you think we need to mention value for timezone as well?

Reply:
Fixed.